src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,23 @@
 package mate.academy;
 
+import mate.academy.lib.Injector;
+import mate.academy.model.Movie;
+import mate.academy.service.MovieService;
+
 public class Main {
-    public static void main(String[] args) {
+    private static final Injector injector = Injector.getInstance("mate.academy");
 
+    public static void main(String[] args) {
+        MovieService movieService = (MovieService) injector.getInstance(MovieService.class);
+        Movie movie1 = new Movie();
```

Comment:
It's better not to use numbers in variable names 
```suggestion
        Movie avengers = new Movie();
```